packages/api/src/services/alertsService.js

```diff
@@ -4,10 +4,15 @@ class AlertsService {
     this.index = 'alerts';
   }
 
-  finalResut(result) {
+  finalResut(result, address) {
     if (result.hasOwnProperty('hits')) {
       if (result.hits.hasOwnProperty('hits')) {
-        return result.hits.hits;
+        return result.hits.hits
+          .map((doc) => ({
+            edition_id: doc._source.edition_id,
+            ...doc._source.rules.find((rule) => rule.address == address)
+          })
+        );
```

Comment:
@m-Peter made some changes in this file, please tell me if they look fine for you

Reply:
Well done :clap: Looks much better now!